src/App.js

```diff
@@ -1,11 +1,33 @@
 //TODO: STEP 1 - Import the useState hook.
-import React from "react";
+import React, {useState} from "react";
 import "./App.css";
 import BottomRow from "./BottomRow";
 
 function App() {
   //TODO: STEP 2 - Establish your applictaion's state with some useState hooks.  You'll need one for the home score and another for the away score.
+  const [homeScore, setHomeScore] = useState(0);
+  const [awayScore, setAwayScore] = useState(0);
 
+  const homeTouchdown = () => {
+    setHomeScore(homeScore + 7);
+  };
+
+  const awayTouchdown = () => {
+    setAwayScore(awayScore + 7);
+  };
+
+  const homeFieldGoal = () => {
+    setHomeScore(homeScore + 3);
+  };
+
+  const awayFieldGoal = () => {
+    setAwayScore(awayScore + 3);
+  };
```

Comment:
good. Although you may even use just one parameteric handler to handle all the scores for both sides - stretch goal